pkg/features/kube_features.go

```diff
@@ -486,7 +486,7 @@ const (
 	EvenPodsSpread featuregate.Feature = "EvenPodsSpread"
 
 	// owner: @matthyx
-	// alpha: v1.16
```

Comment:
keep the alpha line and add the beta line so we can easily see which releases this went into